docs/proposals/202003_thanos_rules_federation.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Global / Federated Rules API
+type: proposal
+menu: proposals
+status: accepted
+owner: s-urbaniak,bwplotka
```

Comment:
```suggestion
owner: s-urbaniak
```

I think essentially this will render wrongly on website (: We need one ;p

Reply:
Happy to co-own this though :+1:

Reply:
Happy to take responsibility for these changes :+1: